### DISCUSSION TITLE
Allow Custom Password Requirements

### DISCUSSION THREAD

**Discussion by Zackptg5** [mealie-recipes/mealie#1878] [Feature Request]
**Before submitting this feature request I have**
- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
I have mealie set as a web facing service behind authelia and haproxy for security. I don't need a secure password to mealie at this point for my user accounts (other than admin).
My ultimate plan is to have this instance be accessible for my whole extended family with a generic user account they can all sign into rather than creating a user account for all 60+ people with a shared group. Since authelia manages the authentication that actually matters, I was hoping to just set an empty password or at least something simple like the same as the user name for the family account.

**Current status**
[With this commit](https://github.com/hay-kot/mealie/commit/3ad3d92e5ffcbb64a547059cf9520acb886caaa2) I'm able to set a custom password, via command line. However, it'd be nice to have a gui option to setup custom requirements or a checkbox to ignore them. 
Also, blank passwords will not work even with the above method (it'll set but it won't let me login)

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by coolkau** [comment]
I am using it in my local network only and would love if I could run this without any login at all.

**Comment by ahhdrrt** [comment]
+1 to this request. 
Using the web page card integration (iframe) in Home Assistant dashboard to display "All Recipes" on my kitchen wall panel. Have to login every time the page changes. Increasing the TOKEN_TIME environment variable only helps if I don't tap on another dashboard card on the screen. Otherwise I have to re-login when I go back to the web page card. 
I can find no successful way to pass authentication information via URL command line since Mealie uses tokens and not Basic authentication (at least I could not discover a way, if anyone has any ideas). 
Assuming it is possible, might we be able to pass the bearer token for a specified user in the URL to permit access?  If that is possible, then perhaps the ability to generate bearer tokens for users from the UI where we can specify their lifetime? Similar to the API key generation already in place. 
Love Mealie, BTW, use it all the time. Just have been wanting to integrate it into home assistant a bit further than just the the Meal of the Day. Thanks for all you have done!

**Comment by bernardovalente** [comment]
+1
I'm in the same situation as @ahhdrrt , the constant need to login makes this unusable within Home Assistant

**Reply by hay-kot** [reply]
FYI If you're using the latest nightly version this should be fixed.
